### PR TITLE
Use the descriptorLanguage pipe to handle Galaxy on the versions tab

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
For Galaxy workflows, display `Galaxy` instead of `gxformat2` in the Versions tab.

TLDR; Use the pipe built just for this.

Longer explanation -- we weren't actually displaying the language version twice; we were displaying the language followed by the language version. For example, for WDL versions it displays something like `WDL draft-2`, where `WDL` is the language and  `draft-2` is the language version.

For Galaxy we made a mistake in having the language be `gxformat2` (not to be confused with the language version); the language is Galaxy. But it got baked in, so we use the pipe to display it as Galaxy, but missed this place.

**Review Instructions**
1. Go to a Galaxy workflow
2. Select the Versions table
3. Ensure that it the language versions say `Galaxy gxformat1` instead of `gxformat2 gxformat1`. Unless you pick the one Galaxy workflow currently in our system that is gxformat2, in which case it should say `Galaxy gxformat2`.

**Issue**
dockstore/dockstore#5701

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

**Review by kathy-t — APPROVED** [review]

**Comment by coverbeck** [comment]
Restructured deletion tests à la Steve's #1857 to hopefully fix integration test that was failing. Worked once. Re-requesting reviews.

**Review by kathy-t — APPROVED** [review]

File: cypress/e2e/group3/deletion.ts [hunk 1]
```diff
@@ -13,75 +13,99 @@
...
+});
+
+describe('Entry Deletion 1', () => {
+  init();
+  deletionTests(entries[1]);
+});
+
+describe('Entry Deletion 2', () => {
+  init();
+  deletionTests(entries[2]);
```
Comment (svonworl) [inline cypress/e2e/group3/deletion.ts:110]: Since we're not iterating through `entries` anymore, we could break out each `Entry` into its own variable (`workflow`, `notebook`, etc) and describe the tests likewise.

```
const notebook = {... }
// ...
describe('Notebook Deletion'), () => {
  init();
  deletionTests(notebook);
}
```

**Review by svonworl — APPROVED** [review]
